src/Razor/src/Microsoft.VisualStudioCode.RazorExtension/Microsoft.VisualStudioCode.RazorExtension.csproj

```diff
@@ -49,6 +49,7 @@
       <Content Include="$(PublishDir)\Microsoft.AspNetCore.Razor.Utilities.Shared.dll" Pack="true" PackagePath="content" CopyToOutputDirectory="PreserveNewest" />
       <Content Include="$(PublishDir)\Microsoft.CodeAnalysis.Razor.Compiler.dll" Pack="true" PackagePath="content" CopyToOutputDirectory="PreserveNewest" />
       <Content Include="$(PublishDir)\Microsoft.CodeAnalysis.Razor.Workspaces.dll" Pack="true" PackagePath="content" CopyToOutputDirectory="PreserveNewest" />
+      <Content Include="$(PublishDir)\Microsoft.CodeAnalysis.Remote.Razor.dll" Pack="true" PackagePath="content" CopyToOutputDirectory="PreserveNewest" />
```

Comment:
It might be better to just package everything and start excluding things instead of including....

Reply:
Or find some way to test that this has the right contents